### ISSUE TITLE
Midisport 8x8 working on OS 13 Ventura?

### ISSUE THREAD

**Issue by antoniograzioli** [leighsmith/midisport-macos#42]
I just replaced my very old iMac 2012 where my Midisport 8x8 was working with a new Mac Mini M2...
The drivers doesn't seem to work, but I suspect that they are actually "blocked" (newest Mac seems to block EVERYTHING)

Has anybody had success with it under OS Ventura? Or do you have any hint to make them work under newest OSs ?
Thanks

**Comment by Chuzzwuzza** [comment]
Yes I have this working on my M2 Mac mini 4x4 Anniversary Edition with Latest Ventura, I did try it a few weeks ago and failed the reason being I didn't have the MIDI_USB_OSX_3.5.3.dmg sat in the download directory I'm guessing thats the reason why.
